### ISSUE TITLE
Invalid JSON data when changing start menu icon

### ISSUE THREAD

**Issue by anventia** [ramensoftware/windows-11-taskbar-styling-guide#124]
I added the following to the bottom of my config JSON:
```json
  "controlStyles[20].target": "Taskbar.ExperienceToggleButton#LaunchListButton[AutomationProperties.AutomationId=StartButton] > Taskbar.TaskListButtonPanel > Border#BackgroundElement",
  "controlStyles[20].styles[0]": "Background:=<ImageBrush Stretch="Uniform" ImageSource="<https://cdn-icons-png.flaticon.com/512/7554/7554093.png>" />",
  "controlStyles[21].target": "Taskbar.ExperienceToggleButton#LaunchListButton[AutomationProperties.AutomationId=StartButton] > Taskbar.TaskListButtonPanel > Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer#Icon",
  "controlStyles[21].styles[0]": "Visibility=Collapsed"
```
however I get "Invalid JSON data" error when trying to save it in Windhawk.
Does anyone know why?

**Comment by bbmaster123** [comment]
without seeing the rest, its most likely because styles have to be in sequential order. if your previous style was for example controlStyles[16].target, the next one would need to be controlStyles[17].target

One other thing to watch out for is to make sure each line, except the very last one, ends with a comma

**Comment by anventia** [comment]
Yeah, previously, the last one as 19, so I added this to the end as 20 and 21. I also made sure to add the comma to the previous last line that didn't have one, and my new last line doesn't have the comma.

**Comment by m417z** [comment]
One hint is that GitHub colors parts of the JSON you posted in red. Quotes inside strings must be escaped - each quote must turn into `\"`.

You can also use one of the online JSON validators.

![image](https://github.com/user-attachments/assets/805724f1-5a60-421d-b29e-17ba4ecb20c7)

Generally, the JSON wasn't designed to be modified by hand. It was designed to backup and share mod settings. Why not use the settings UI instead?

**Comment by bbmaster123** [comment]
ah of course, good catch
Yes, that style in JSON should be:
`"controlStyles[20].styles[0]": "Background:=<ImageBrush Stretch=\"Uniform\" ImageSource=\"<https://cdn-icons-png.flaticon.com/512/7554/7554093.png>\" />",`

**Comment by anventia** [comment]
Thanks! It resolved the error, However, now my Start Button icon is completely gone, it didn't seem to replace the image properly.
```json
"controlStyles[20].target": "Taskbar.ExperienceToggleButton#LaunchListButton[AutomationProperties.AutomationId=StartButton] > Taskbar.TaskListButtonPanel > Border#BackgroundElement",
    "controlStyles[20].styles[0]": "Background:=<ImageBrush Stretch=\"Uniform\" ImageSource=\"https://cdn-icons-png.flaticon.com/512/7554/7554093.png>\" />",
    "controlStyles[21].target": "Taskbar.ExperienceToggleButton#LaunchListButton[AutomationProperties.AutomationId=StartButton] > Taskbar.TaskListButtonPanel > Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer#Icon",
    "controlStyles[21].styles[0]": "Visibility=Collapsed"
```

I followed the instructions [here](https://github.com/ramensoftware/windows-11-taskbar-styling-guide?tab=readme-ov-file#start-button-image), and also tried the image links shown as examples in case it was a problem with the image I was using, but for some reason it doesn't show up.
![image](https://github.com/user-attachments/assets/1c2ff3d2-d91b-4b5d-b292-22eee5dcf109)

**Comment by bbmaster123** [comment]
just a small typo in your ImageSource, you have `.png>\"` when it should be just `.png\"`
